python_modules/dagster/dagster/_core/definitions/instigation_logger.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from contextlib import ExitStack
+from typing import IO, List, Optional
+
+from dagster import _seven
+from dagster._core.instance import DagsterInstance
+from dagster._core.storage.captured_log_manager import CapturedLogManager
+from dagster._core.utils import coerce_valid_log_level
+from dagster._utils.log import create_console_logger
+
+
+class InstigationLogHandler(logging.Handler):
```

Comment:
If I'm understanding correctly, the functionality here isn't specific to Instigation logs. Thoughts on just calling it `DispatchingLogHandler` or `ForwardingLogHandler` or something?